app/HutchManager/Services/UserService.cs

```diff
@@ -125,6 +125,7 @@ public async Task Create(UserModel userModel)
       Email = userModel.Email
     };
     await _users.CreateAsync(user);
```

Comment:
I suppose links to my comment above. If this fails, you need to work out why and send the appropriate code back. If it just dies because the db is shut down or inaccessible, but the user did the right thing, send 500 (internal server error). If the user sent something that causes it not to work, you need to send back 400 (bad request)

Reply:
as above, regardless of the framework automagically helping, it's not the service's job to talk in http status codes.

Services should be abstracted from the context in which they're used, so they can be reused.

So e.g. if you ask a Service for a single item from a db, for example, and it doesn't find anything, it should return null, or false, or a result object that clearly indicates no success.

Then if you call that service in a web app, yes the calling code should ultimately check for that result and return 404.

but if you call that service in a console app, what use is a 404? you probably want to check the result and then print a message to the user on stdout or stderr.

You get the idea. Services should be reusable. It's the same reason we have them accept and return models, not entities.